templates/triggers/trigger_schedule_in_batch.haml

```diff
@@ -65,21 +79,21 @@
       });
     }
 
-    function buildFormData(form) {
-      return form.serializeArray().reduce(function(acc, item) {
-          let value = item.value;
-          if (item.name === 'omnibox' && value) {
-            value = acc['omnibox'] || [];
-            value.push(item.value);
-          }
-          acc[item.name] = value;
-          return acc;
-        }, {});
-    }
-
     document.addEventListener("DOMContentLoaded", function() {
       const formContainer = document.querySelector('#id-trigger-schedule-in-batch');
       const repeatPeriod = formContainer.querySelector("#id_schedule_repeat_period");
+      const tembaField = formContainer.querySelector('#id_schedule_omnibox').shadowRoot
+        .querySelector('temba-select').shadowRoot
+        .querySelector('temba-field');
+
+      const helpTextNode = document.createElement('div');
+      const textNode = document.createTextNode('The batches will deploy in the order you add the group');
+      helpTextNode.appendChild(textNode);
+      helpTextNode.setAttribute('style', 'font-size: 11px; margin-top: 6px; opacity: 1;');
+      tembaField.appendChild(helpTextNode);
+      const label = tembaField.shadowRoot.querySelector('label');
+
+      if (label.innerHTML === '<!----><!---->') label.innerHTML = 'Groups';
```

Comment:
`label.innerText === ""` is safer condition here

Reply:
I'll just remove the condition all together as it serves no clear purpose. Any way using `label.innerText === ""`  will always return false so the label never gets updated as expected.